src/useDebouncedCallback.ts

```diff
@@ -57,8 +56,8 @@ export default function useDebouncedCallback<T extends (...args: any[]) => Retur
   const startTimer = useCallback(
     (pendingFunc, wait) => {
       if (useRAF) {
-        window.cancelAnimationFrame(timerId.current);
-        return window.requestAnimationFrame(pendingFunc);
+        cancelAnimationFrame(timerId.current);
+        return requestAnimationFrame(pendingFunc);
       }
```

Comment:
Your `useRAF` condition includes `typeof window !== 'undefined'` subcondition.

```js
const useRAF = !wait && wait !== 0 && typeof window !== 'undefined';
```

It means that the script is running in the browser and global namespace is `window`, so there is no need to prepend `requestAnimationFrame` with `window.`